src/index.jsx

```diff
@@ -39,30 +46,39 @@ const App = () => {
 
   return (
     <AppProvider store={initializeStore()}>
-      <Head />
-      <Switch>
-        <Route path="/home">
-          <StudioHome />
-        </Route>
-        <Route
-          path="/course/:courseId"
-          render={({ match }) => {
-            const { params: { courseId } } = match;
-            return (
-              <CourseAuthoringRoutes courseId={courseId} />
-            );
-          }}
-        />
-        <Route
-          path="/course_rerun/:courseId"
-          render={({ match }) => {
-            const { params: { courseId } } = match;
-            return (
-              <CourseRerun courseId={courseId} />
-            );
-          }}
-        />
-      </Switch>
+      <QueryClientProvider client={queryClient}>
+        <Head />
+        <Switch>
+          <Route path="/home">
+            <StudioHome />
+          </Route>
+          <Route
+            path="/course/:courseId"
+            render={({ match }) => {
+              const { params: { courseId } } = match;
+              return (
+                <CourseAuthoringRoutes courseId={courseId} />
+              );
+            }}
+          />
+          <Route
+            path="/course_rerun/:courseId"
+            render={({ match }) => {
+              const { params: { courseId } } = match;
+              return (
+                <CourseRerun courseId={courseId} />
+              );
+            }}
+          />
+          {process.env.ENABLE_TAGGING_TAXONOMY_PAGES === 'true' && (
```

Comment:
We don't necessarily need a build-time switch to disable this functionality, as long as the waffle flag prevents any users from seeing it until we turn the waffle flag on for them. But it might be a good idea to have for the first sprint or two while this is super new; not sure.

Reply:
^ Actually, ignore this comment... I see many other functionality like this has similar build-time switches.

Reply:
But please add `ENABLE_TAGGING_TAXONOMY_PAGES = false` as a default to `.env` and `.env.development`, and set it to `true` in `.env.test`

Reply:
> But please add ENABLE_TAGGING_TAXONOMY_PAGES = false as a default to .env and .env.development, and set it to true in .env.test. Maybe you could even set it true in .env.development - there's no harm in making it easy for developers to set up and test this :)

Sure :+1: 

> ^ Actually, ignore this comment... I see many other functionality like this has similar build-time switches.

I think it is because you can enter the page through the url regardless of whether the waffle flag is disabled

Reply:
Added here https://github.com/openedx/frontend-app-course-authoring/pull/622/commits/9ec01ce2d6bfeb103b630dc86d569fceb1f153b3